### ISSUE TITLE
FedEx vs Fedex

### ISSUE THREAD

**Issue by mdlutz24** [VolantisDev/drupal-commerce-fedex#5]
This is a note, mostly for myself. FedEx is short for Federal Express, so in camelCase and anytime the F is capitalized, I'm going to capitalize the E as well. I'm not sure I've been doing it consistently, but I think it should be FedExSomething, fedExSomething, fedex_something, but never FedexSomething.  I'm going to go with that from now on, and fix any previous mistakes as I go.

**Comment by ReplicaObscura** [comment]
Agreed from my end. I've been trying to do it that way, but it's possible I've made some mistakes in that regard as well. I'll keep this in mind moving forward.
